differ/utils.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2022 Red Hat, Inc
```

Comment:
httpUtils perhaps? (it's still bad, but not as bad as `utils` or `common` :)

Reply:
Looked into the insights-operator-utils, could not find these in there.